test/unit/spec/PollController_spec.js

```diff
@@ -108,13 +102,8 @@ function (Okta, OktaAuth, Util, PollingForm, Expect, Router, $sandbox, resPollin
               }
             });
             done();
-          }, 9000);
+          }, 5000);
```

Comment:
certainly not idea to use real timeout in test. what's the challenge to mock time?

Reply:
@haishengwu-okta I have tried using jasmine.clock(). If we do that we'll see this error. 
```  
debug.js:21 Error: Jasmine Clock was unable to install over custom global timer functions. Is the clock already installed?
    at <Jasmine>
    at UserContext.<anonymous> (PollController_spec.js:91)
    at <Jasmine>

```
uninstalling before installing the clock also does not resolve the problem.

@aarongranick-okta has a current PR to fix these issues https://github.com/okta/okta-signin-widget/compare/4.0?expand=1#diff-8ef46a6d97cdf70fb7322bc2ef22b3f8R340 and I was hoping to avoid making all these changes here

Reply:
oh, I see. Thanks. Looks like it might be some mess of Jasmine Clock somewhere but yeah, it's hard to figure it out.